api/graphql/schema.graphql

```diff
@@ -1396,22 +1414,26 @@ type Membership {
   # The agreements this user has agreed to (or not) for this group
   agreements: MembershipAgreementQuerySet
   createdAt: Date
+  # The common roles this member has in this group
+  commonRoles: CommonRoleQuerySet
   groupId: ID
   group: Group
-  # Is this group membership a moderator role
+  # Is this group membership a moderator role. TODO: remove once mobile app has been updated to use new roles
   hasModeratorRole: Boolean
   # Answers to the join questions for this group
   joinQuestionAnswers: GroupJoinQuestionAnswerQuerySet
   # The last time the current logged in user viewed this group (if they are a member of it)
   lastViewedAt: String
+  # The common roles this members has in this group - just the join table data
+  membershipCommonRoles: MembershipCommonRoleQuerySet
+  # The group defined roles this members has in this group - just the join table data
+  membershipGroupRoles: MembershipGroupRoleQuerySet
   # Number of new posts in this group since the current logged in user viewed it (if they are a member of it)
   newPostCount: Int
   person: Person
-  # 0: default, 1: moderator
+  # 0: default, 1: moderator. TODO: deprecated, remove once mobile app has been updated to use new roles
```

Comment:
A bunch of the mobile graphQL queries request role but they don't actually use them. They were just synced up with their EVO counterparts. But I guess if you remove role here, it will still bomb those queries... :(